indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/BagsIndexConfig.scala

```diff
@@ -0,0 +1,14 @@
+package uk.ac.wellcome.platform.archive.indexer.bags
+
+import com.sksamuel.elastic4s.ElasticDsl.{dateField, keywordField}
+import com.sksamuel.elastic4s.requests.mappings.FieldDefinition
+import uk.ac.wellcome.platform.archive.indexer.elasticsearch.IndexConfig
+
+object BagsIndexConfig extends IndexConfig {
+  override protected val fields: Seq[FieldDefinition] =
+    Seq(
+      keywordField("id"),
+      dateField("createdDate"),
+      keywordField("type")
```

Comment:
Do we need to type these objects? We have for the ingests, but that's mostly a holdover from the display models, and I'm not sure it's useful.

Reply:
I'm not sure - i'll think about that more thoroughly when we apply the proper mappings.